src/comonents/ButtonPage.jsx

```diff
@@ -1,9 +1,11 @@
-import React from 'react'
+import React, { useContext } from 'react'
+import { MyNoteContext } from '../context/NoteState'
 
 function ButtonPage() {
```

Comment:
this component name should be  Button, not page,
this represent particular button component.

Reply:
Changed sir